ravenframework/Models/PostProcessors/SparseSensing.py

```diff
@@ -52,10 +54,26 @@ class cls.
                                                 printPriority=108,
                                                 descr=r"""Features/inputs of the data model""")
     goal.addSub(features)
-    target = InputData.parameterInputFactory("target", contentType=InputTypes.StringType,
+    xValue = InputData.parameterInputFactory("xValue", contentType=InputTypes.StringListType,
```

Comment:
Note that changing the input usually requires an automated script be created and placed in scripts/conversionScripts/